src/hotspot/share/code/SCCache.cpp

```diff
@@ -1131,6 +1164,20 @@ bool SCCache::finish_write() {
         } else if (_load_entries[i].for_preload() && _load_entries[i].method() != nullptr) {
           // record entrant first version code for pre-loading
           preload_entries[preload_entries_cnt++] = entries_count;
+        } else if (entries_address[i].kind() == SCCEntry::Adapter) {
+          adapters_count++;
+        } else if (entries_address[i].kind() == SCCEntry::Stub) {
+          stubs_count++;
+        } else if (entries_address[i].kind() == SCCEntry::Blob) {
+          total_blobs_count++;
+          if (entries_address[i].comp_level() == CompLevel_none) {
+            shared_blobs_count++;
+          } else if (entries_address[i].comp_level() == CompLevel_simple) {
```

Comment:
we also caching C1 code with simple profiling: `CompLevel_limited_profile`

Reply:
Added check for `CompLevel_limited_profile`